convert2rhel/utils.py

```diff
@@ -472,9 +472,18 @@ def download_pkg(
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
         loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
+            " you are using an env that overrides an inhibitor and"
+            " you understand that if you continue you might lose the rollback."
+            " \nOutput from the yum downloader call:\n%s %s" % (pkg, output)
         )
+
+        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:  # template for what needs to be done
+            loggerinst.critical("")
+        loggerinst.info(
+            "could not complete the rollback because one or more files are not updated, Check to see"
+            "if all packages are updated and try the conversion again"
+        )
```

Comment:
Rollback isn't happening here. It's just that when the rollback happens and they have the override variable active the rollback won't have the package backed up. I'd also change this to a warning

Reply:
In the new code for this I have totally changed the message this warning outputs